content/windows/docs/oom.md

```diff
@@ -34,8 +34,12 @@ In the example below, when the Kubernetes scheduler tries to place a pod on a no
         memory: 800Mi
       requests:
         cpu: .1
-        memory: 128Mi
+        memory:
 ```
+**Note:**
+*When you set memory resource limits for Windows containers, you should either set a limit and leave the memory request unspecified, or set the request equal to the limit.*
```

Comment:
Why would you leave requests blank? Requests are used for scheduling pods. If it's not set, the scheduler may try running the pod on a node that doesn't have enough system resources.

Reply:
On Windows, it was identified an issue when setting both options where the "request" become the **"limit"**. The Kubernetes-Sig has been working to fix it and I'll update our best practice as soon as we have it fixed.

The Windows-SIG also added the same info in the official documentation.

Reply:
Still not fixed.